error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -144,6 +144,28 @@ Mono<S> after(Mono<T> mono, S object) {
     }
   }
 
+  /**
+   * Prefer {@link Flux#take(long, boolean)} over {@link Flux#take(long)}.
+   *
+   * <p>In Reactor versions prior to 3.5.0, `Flux#take(long)` makes an unbounded request upstream,
+   * and is equivalent to `Flux#take(long, false)`. In 3.5.0, the behavior of `Flux#take(long)` will
+   * change to that of `Flux#take(long, true)`.
+   *
+   * <p>The intent with this Refaster rule is to get the new behavior before upgrading to Reactor
+   * 3.5.0.
+   */
+  static final class FluxTake<T> {
+    @BeforeTemplate
+    Flux<T> before(Flux<T> flux, long n) {
+      return flux.take(n);
+    }
+
+    @AfterTemplate
+    Flux<T> after(Flux<T> flux, long n) {
+      return flux.take(n, /* limitRequest= */ true);
```

Comment:
IMO, the comment doesn't add much. The description is already clear enough.
```suggestion
      return flux.take(n, true);
```

Reply:
This is suggested by the `BooleanParameter` check (see comment above) :)